src/dev/storage/platform/M24C32.cpp

```diff
@@ -25,21 +25,36 @@ uint32_t M24C32::readWord(uint32_t address) {
 }
```

Comment:
writeHalfWord(), readHalfWord(), writeWord(), and readWord() all have the potential to cause the same problem as reading/writing the arrays did, so they should call readBytes() instead of i2c.readMemReg().

Reply:
Good catch! Those now use the read/write bytes